app/dashboard/models.py

```diff
@@ -429,6 +438,39 @@ def get_natural_value(self):
         decimals = token.get('decimals', 0)
         return float(self.value_in_token) / 10**decimals
 
+    @property
+    def no_of_applicants(self):
+        return self.interested.count()
+   
+    @property
+    def has_applicant(self):
+        """Filter results by bounties that have applicants."""
+        return self.prefetch_related('activities') \
+            .filter(
+                activities__activity_type='worker_applied',
+                activities__needs_review=False,
+            )
+        
+    @property
+    def warned(self):
+        """Filter results by bounties that have been warned for inactivity."""
+        return self.prefetch_related('activities') \
+            .filter(
+                activities__activity_type='bounty_abandonment_warning',
+                activities__needs_review=True,
+            )
+    
+    @property
+    def escalated(self):
+        """Filter results by bounties that have been escalated for review."""
+        return self.prefetch_related('activities') \
+            .filter(
+                activities__activity_type='bounty_abandonment_escalation_to_mods',
+                activities__needs_review=True,
+            )
+
+
+
```

Comment:
@kuhnchris extra lines need to be removed

Reply:
I wonder, since usually lint should hit my shins and makes travis fail whenever I do more than 2 lines? 😕 I can remove those two line tho if you want. I add them usually to indicate blocks, but that one looks just... stupid, correct. I'll fix that, sec.